integration_tests/class_01.py

```diff
@@ -0,0 +1,28 @@
+from lpython import i32,f64
+from math import sqrt
+
+class coord:
+    def __init__(self: "coord"):
+        self.x: i32 = 3
+        self.y: i32 = 4
```

Comment:
Currently, this is represented as:
```py
class coord:
	self: "coord"
	x: i32 = 3
	y: i32 = 4
```

And @tanay-man wants this PR to be merged and work on the arguments initialisation in the next PR.